cc_licenses/templates/includes/legalcode_10_license.html

```diff
@@ -0,0 +1,68 @@
+{% load bidi i18n license_tags %}
+<div id="legal-code-body" class="padding-larger margin-top-bigger has-text-black">
+  {% include 'includes/snippet/icon_header_snippet.html' %}
+  <div>
+    <h3 class="padding-bottom-normal b-header">
+      {% blocktrans trimmed %}CC0 1.0 Universal{% endblocktrans %}{# Long full title #}
+    </h3>
+    <p class="body-big padding-bottom-larger">
+{% blocktrans trimmed %}CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES NOT PROVIDE LEGAL SERVICES. DISTRIBUTION OF THIS DOCUMENT DOES NOT CREATE AN ATTORNEY-CLIENT RELATIONSHIP. CREATIVE COMMONS PROVIDES THIS INFORMATION ON AN "AS-IS" BASIS. CREATIVE COMMONS MAKES NO WARRANTIES REGARDING THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS PROVIDED HEREUNDER, AND DISCLAIMS LIABILITY FOR DAMAGES RESULTING FROM THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS PROVIDED HEREUNDER.{% endblocktrans %}</p>
+
+      <div class="padding-bottom-larger">{# section - SOP #}
+ <p id="s1" class="body-bigger padding-bottom-normal"><strong>{% blocktrans trimmed %}Statement of Purpose{% endblocktrans %}</strong></p>
```

Comment:
We need to update the side-menu component so that it has anchor tags and section titles for this new legalcode section when zero licenses are being viewed.

Reply:
Ah, I missed that. Will do.